pyproject.toml

```diff
@@ -3,4 +3,5 @@ requires = ["setuptools", "wheel", "Cython"]
 
 [tool.black]
 line-length = 120
-skip-string-normalization = true
+skip-string-normalization = false
+exclude = '^/testsuite'
```

Comment:
Should probably be `extend-exclude` to avoid removing the default exclusions.

Reply:
That did not work at all for me locally. Not sure why. Is black outdated for us?

Reply:
Works fine under 21.8b0 on Debian 11, but not 19 or whatever the version we have in the workflow is. What do we want to do?

Reply:
Yes. Try upgrading to `21.8b0` and see if it breaks anything. If not, let's just upgrade.